src/index.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <App />
+    </Provider>
   </React.StrictMode>,
 );
 
```

Comment:
- [ ] Kindly make sure that your project is running properly. When i try to run the app, I get the following error message preventing me from accessing your app.
![image](https://user-images.githubusercontent.com/62290545/207909631-b012a6ba-5860-4045-9f80-b606b76a3dcf.png)


- [HINT] The error is caused by the following piece of code in the `NavBar` component.
https://github.com/Enning94/Bookstore-project/blob/2a8323945ffcfa25509404163146680ad7c76fc3/src/Components/NavBar.js#L2
- The error indicates that it can't find the path to a folder called "Styles" which doesn't exist.
![image](https://user-images.githubusercontent.com/62290545/207910199-7e310f32-40c5-42c5-8ca2-a04cfce55e35.png)
- When I searched in your code for a bit, I found that in your `.gitignore` file you're excluding the "Styles" directory which then GitHub ignores.
https://github.com/Enning94/Bookstore-project/blob/2a8323945ffcfa25509404163146680ad7c76fc3/.gitignore#L25-L26

Reply:
Thank you for such descriptive feedback. 👍